### DISCUSSION TITLE
Production ready ?

### DISCUSSION THREAD

**Discussion by alxlion** [braverhealth/phoenix-socket-dart#43] [General]
Hi ! 

First, thanks @matehat for this project and for continuing to maintain it. 

I have a Phoenix backend and I'd like to use it on a Flutter app for mobile. Do you have projects running with this dependency ? Is the presence channel working well ?

Thanks for your feedback !

**Comment by matehat** [comment]
Hello @alxlion !

It is indeed being used in production (iOS, Android and Web) and we've not seen any reliability issues for quite a while! 

We (at Braver) do not use Phoenix Presence at the moment, though. This part was contributed by @ol88 so I'll let him/her comment about its reliability.

Thanks for your kind words!

**Comment by ol88** [comment]
Hi @alxlion

As far as I'm concerned, Presence works well for my needs and I use it all the time without issue during dev but I don't have anything in production yet so can't vouch for full stability at this stage.

Either way, it's a pretty simple module and I think it should run in production just fine. It's actually just a dart port of Phoenix's own Javascript Presence client, it was not written from scratch. If you have any issue with it, a quick PR should do the trick. @matehat is great and responsive to integrate any sensible changes/bug fix.

**Comment by alxlion** [comment]
Thanks guys for your answers ! 💯

**Comment by acrolink** [comment]
@matehat 

I just want to say thank you. I am impressed there is already a library to connect to `Phoenix` channels in `Dart`. Looking forward to start using it. Great work!

**Reply by matehat** [reply]
Thank you for your kind words!

If you encounter any issues, don't hesitate to report them!

**Comment by jaybe78** [comment]
Hi Guys,

First thanks @matehat and everybody who has=ve taken time to develop the client so that we can take advantage of the incredible phoenix channels in our flutter apps.

From what you  said @matehat, it's robust enough for a production app.
I would like to know if you still actively maintain the lib and if you plan on making more improvements in the future.

I can see that that the lib relies on 
web_socket_channel 2.0.0

Are you gonna upgrade it to ^3.0.1. It would not hurt I guess

**Reply by matehat** [reply]
Hi 👋 

Our team and I, at Braver, are still heavily using it in production, so we're always open to add new features and fix any issues that come up.

Hope that helps!

For the web_socket_channel, I'm open to look into this, please file an issue.
